projects/ng-event-plugins/src/plugins/silent.plugin.ts

```diff
@@ -14,7 +14,7 @@ export class SilentEventPlugin extends AbstractEventPlugin {
         event: string,
         handler: Function,
     ): Function {
-        SilentEventPlugin.ngZone = this.manager.getZone();
+        SilentEventPlugin.ngZone = SilentEventPlugin.ngZone ?? this.manager.getZone();
```

Comment:
Why do we even have this? Why not just `this.manager.getZone().runOutsideAngular(...)`? Probably something from old times, let's drop this trick completely.

Reply:
What do we do for?

<img width="935" alt="image" src="https://github.com/user-attachments/assets/65859f97-9110-4b96-9fdb-edd50ccb06f5">